Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -120,6 +120,20 @@ public let CodeGenerators: [CodeGenerator] = [
         b.createTemplateString(from: parts, interpolating: interpolatedValues)
     },
 
+    CodeGenerator("CallTaggedTemplateGenerator", input: .function()) { b, f in
```

Comment:
I'm wondering if this code generator should also generate a random tag function  similar to the other function generators, with something like
```swift
            b.generateRecursive()
            b.doReturn(value: b.randVar())
```
as body. The main benefit I see is that the tag function could be given the appropriate signature here (first arg: array of string, remaining args: anything), because we know it'll be used as a tag. The `InputMutator` will take care of using other functions as tags eventually. Let me know what you think!

Reply:
Yeah that's a good idea! I'll drop the current input to the generator in that case and add a tag generator.